packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -295,6 +296,13 @@ class CheckboxListTile extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
+  /// Whether the CheckboxListTile is interactive.
+  ///
+  /// If false, this list tile is styled with the disabled color from the
+  /// current [Theme] and the [onTap] callback is
```

Comment:
```suggestion
  /// current [Theme] and the [ListTile.onTap] callback is
```
I think this change got reverted somehow.

Reply:
Thanks